src/chrome/content/rules/Fefe.xml

```diff
@@ -31,17 +30,23 @@
 	^fefe.de doesn't exist.
 
 -->
+<ruleset name="Fefe.de (partial)" platform="Lets Encrypt">
+  <target host="blog.fefe.de" />
+        <target host="ptrace.fefe.de" />
+        <target host="www.fefe.de" />
+
+        <rule from="^http:"
+                to="https:" />
+</ruleset>
+
 <ruleset name="Fefe.de (partial)" platform="cacert" default_off="missing certificate chain">
```

Comment:
to re-enable this ruleset, remove `platform` & `default_off` argument, so it's just `<ruleset name="Fefe.de (partial)"`.